android/sample_app/src/main/java/app/cash/trifle/MainActivity.kt

```diff
@@ -10,7 +10,11 @@ class MainActivity : AppCompatActivity() {
     super.onCreate(savedInstanceState)
 
     val version = LibraryVersion()
-    Log.v(TAG, "Security SDK version: $version")
+    Log.v(TAG, "Trifle version: $version")
+
+    //Example Trifle use.  Shouldn't typically be on the main UI thread, but that's the only reason
+    // we're here so it's ok.
+    val keyHandle = KeyHandle.generateKeyHandle("alias")
```

Comment:
Is alias the same as key tag  in ios? if so, shall we converge on one name?
also in ios we decided on having a uuid associated with each tag, see https://github.com/cashapp/trifle/blob/f346fed3d0867636939808a495d52ddf1d7ca813/ios/Trifle/Sources/Digital%20Signature/Secure%20Enclave/SecureEnclaveDigitalSignatureKeyManager.swift#L39

we probably want to do the same here? or you can simply have it as a todo, so we dont forget.

Reply:
btw I put this comment here but it should be for line 15 of the next file! sorry..

Reply:
Added a TODO (and will add a ticket) to deliberately deal with the iOS/Android discrepancy.